README.md

```diff
@@ -111,6 +111,15 @@ To see a step-by-step example, please see our example [here](./EXAMPLE.md)
 
 `-vv, --verbose --verbose` more verbose messages
 
+## APerf Issues?
+Below are some prerequisites for profiling with APerf:
+1. Select the [appropriate instance size](https://github.com/aws/aws-graviton-getting-started/blob/main/perfrunbook/debug_hw_perf.md) if you need PMU stats.
+2. For collecting PMU counter metrics w/o `root` or `sudo` permissions, set the `perf_event_paranoid` to `0`.
+3. To collect PMU counter metrics, APerf needs to open up to 50 file descriptors per vCPU. So, increase `ulimit` settings accordingly.
+4. APerf needs access to `/proc/kallsyms`, so we need to relax `kptr_restrict` by setting it to `0` (on Ubuntu OS).
+5. To enable function-level profiling, install the `perf` binary on your instances.
```

Comment:
For Ubuntu, I recommend installing perf as follows:
```
sudo apt-get install linux-tools-`uname -r`
```

Because AMI may not track the "linux-tools-generic" that would get installed otherwise.

I'm not sure what the right command on AL2 is.

I personally find it helpful to keep these in a little code block so I can copy-paste them all at the same time:
```
sudo apt-get install linux-tools-`uname -r` -y
echo 0 | sudo tee /proc/sys/kernel/kptr_restrict
echo 0 | sudo tee /proc/sys/kernel/perf_event_paranoid
echo 100  | sudo tee /sys/devices/cpu/perf_event_mux_interval_ms
```

Reply:
I left it open and let the users intall `perf` using the commands on the respective OS.
I like the code block copy-paste idea but it might not work for all OS. Hence, I didn't post it either.